docs/docs-content/clusters/cluster-management/kubernetes-dashboard.md

```diff
@@ -50,23 +51,45 @@ process.
 
 :::
 
-10. No customization is required for this deployment. Select **Confirm & Create** on the pack customization screen.
+9. Leave the default values for the **Spectro Kubernetes Dashboard** pack and select **Confirm & Create** on the pack
+   customization screen.
 
-11. The profile overview screen reflects that the `spectro-k8s-dashboard` pack is now part of your cluster profile.
+10. The profile overview screen reflects that the `spectro-k8s-dashboard` pack is now part of your cluster profile.
     Select **Save Changes**.
 
-12. From the left main menu, select **Clusters**. Select the cluster you wish to update.
+11. From the left main menu, select **Clusters**. Select the cluster you wish to update.
 
-13. Select the **Profile** tab.
+12. Select the **Profile** tab.
 
-14. From the applicable **Infrastructure Layers** or **Addon Layers** version drop-down menu, select the profile version
+13. From the applicable **Infrastructure Layers** or **Addon Layers** version drop-down menu, select the profile version
     that contains the **Spectro Kubernetes Dashboard** pack.
 
-15. **Save** your changes.
+14. Click **Save Changes**.
 
 ![Image showing the selection of the profile version to be applied to the selected cluster](/clusters_cluster-management_spectro-kubernetes-dashboard_apply-profile.webp)
 
-16. Use the **Overview** tab to monitor the cluster's status until the **Spectro Kubernetes Dashboard** layer is
+15. Select **Settings** from the top right, and click on **Cluster Settings**.
+
+16. Click on **RBAC**, and on the Cluster tab, click **Add New Binding**.
+
+17. On the **Add Cluster Role Binding**, enter `cluster-admin` for **Cluster Role name**.
+
+18. Leave the **Subject type** set to user, and enter the email address of the user in **Subject Name**. Click
+    **Confirm**.
+
+![Image showing how to create a Cluster Role Binding](/clusters_cluster-management_spectro-kubernetes-dashboard_add-role-binding.webp)
+
+19. Repeat this step as needed.
+
+20. Click the **X** at the top left to close the **Settings** page.
```

Comment:
```suggestion
20. Close the **Settings** page.
```

Reply:
This doesn't have to be accepted, as it's more of a personal preference. I feel like we shouldn't need to be so explicit here.